file.c

```diff
@@ -6523,7 +6523,7 @@ const char ruby_null_device[] =
  *
  *  Note that file permissions are quite different from the _mode_
  *  of a file stream (\File object).
- *  See {IO Modes}[#class-IO-label-Modes].
+ *  See {IO Modes}[IO.html#class-IO-label-Modes].
```

Comment:
RDoc provides link notation to sections, as well as methods.
```suggestion
 *  See IO@Modes.
```
I think this is much preferable than the generated file name and section name.